### PR TITLE
BaSP-TG-44 AND 48: timesheet GET and DELETE tests 

### PR DESCRIPTION
**Pull request by RicardoDosko**
Timesheets - Testing GET controllers.
Timesheets - Testing DELETE controllers.

### CODE REVIEW

File: src/tests/time-sheets.test.js [hunk 1]
```diff
@@ -1 +1,339 @@
...
+        endDate: '2022-07-07T00:00:00.000+00:00',
+      },
+    );
+
+    expect(response.body.msg).toMatch('Code 400: No employee with the id');
+  });
+
+  // INVALID DATA
+
+  test('Ivalid task id', async () => {
```
Comment (dayanagm96) [inline src/tests/time-sheets.test.js:99]: *Invalid. Line 117,135, and 207 as well

File: src/tests/time-sheets.test.js [hunk 2]
```diff
@@ -1 +1,339 @@
...
+        projectId: '6283b662e53ed4c648db8a45',
+        startDate: '2022-06-05T00:00:00.000+00:00',
+        endDate: '2022-07-07T00:00:00.000+00:00',
+      },
+    );
+
+    expect(response.body.msg).toBe('Code 400: Timesheet already exists');
+  });
+
+  // PATH'S OBJECT ID NOT FOUND IN THE CORRESPONDING COLLECTION
```
Comment (dayanagm96) [inline src/tests/time-sheets.test.js:59]: Remember to delete the comments. Line 225 as well

File: src/tests/time-sheets.test.js [hunk 3]
```diff
@@ -1 +1,339 @@
...
+          '62891835d4e286802a02756e',
+          '62891835d4e286802a02756f',
+          '62891835d4e286802a02756g',
+        ],
+        projectId: '6283b662e53ed4c648db8a45',
+        startDate: '2022-06-05T00:00:00.000+00:00',
+        endDate: '2022-07-07T00:00:00.000+00:00',
+      },
+    );
+    // expect(response.body.msg).toBe('Code 400: Timesheet already exists');
```
Comment (dayanagm96) [inline src/tests/time-sheets.test.js:257]: Why is this commented?

File: src/tests/time-sheets.test.js [hunk 4]
```diff
@@ -1 +1,339 @@
...
+    const response = await request(app).delete(`/timesheets/${idCatcher}`).send();
+    expect(response.status).toEqual(204);
+  });
+  test('Should not delete a proyect ', async () => {
+    const response = await request(app).delete('/timesheets/6274613d5f1b9c4131f527e4').send();
+    expect(response.status).toEqual(404);
+  });
+});
+
+// expect(response.body.msg).toBe('There is no timesheet with this Id 6274613d5f1b9c4131f527e4');
```
Comment (dayanagm96) [inline src/tests/time-sheets.test.js:339]: What's this?

File: src/tests/time-sheets.test.js [hunk 5]
```diff
@@ -1 +1,339 @@
...
+  });
+  test('The path is wrong', async () => {
+    const response = await request(app).get('/time/sheets').send();
+    await expect(response.status).toBe(404);
+  });
+  test('The path is wrong', async () => {
+    const response = await request(app).get('/time-sheets').send();
+    await expect(response.status).toBe(404);
+  });
+  test('The path is wrong', async () => {
```
Comment (dayanagm96) [inline src/tests/time-sheets.test.js:295]: Is it needed to have the same test three times in a single function?

**Review by dayanagm96 — APPROVED** [review]

**Review by lucianofrailelopez — APPROVED** [review]

**Review by lucasrendo — APPROVED** [review]

**Pull request merged**